src/ui/components/MetadataCard/index.js

```diff
@@ -25,10 +25,13 @@ const MetadataCard = (props: Props) => {
           throw new Error('title is required');
         }
 
+        const hasContent = content || content === '' ||
```

Comment:
This was hard for me to understand. It would be nice if `metadata` was [typed properly](https://github.com/willdurand/addons-frontend/blob/a3ecc0d751b6bd5dc7e55c4096d9cb5c5ccb6430/src/ui/components/MetadataCard/index.js#L10).

Reply:
This is not a change request, just a comment :)